specification.markdown

```diff
@@ -245,6 +245,68 @@ A `Binary Element` MUST declare a length in octets from zero to `VINTMAX`.
+
+| Status      | Element ID | Element Data Size | Element Data       |
+|-------------|------------|-------------------|--------------------|
+| Before edit | 0x3B4040   | 0x84              | 0x65626d6c         |
+| After edit  | 0x3B4040   | 0x4003            | 0x6d6b76           |
+
+This method is only RECOMMENDED for reducing `Element Data` by a single octet; for reductions by two or more octets it is RECOMMENDED to fill the freed space with a `Void Element`.
+
+Note that if the `Element Data` length needs to be rewritten as shortened by one octet and the `Element Data Size` could be rewritten as a shorter `VINT` then it is RECOMMENDED to rewrite the `Element Data Size` as one octet shorter, shorten the `Element Data` by one octet, and follow that `Element` with a `Void Element`. For example, the following table depicts a `String Element` that stores an `Element ID` (3 octets), `Element Data Size` (2 octets, but could be rewritten in one octet), and `Element Data` (3 octets). If the `Element Data` is to be rewritten to a two octet length, then another octet can be taken from `Element Data Size` so that there is enough space to add a two octent `Void Element`.
+
+Status | Element ID | Element Data Size | Element Data | Void Element 
+-------|------------|-------------------|--------------|-------------
+Before | 0x3B4040   | 0x4003            | 0x6d6b76     |
+After  | 0x3B4040   | 0x82              | 0x6869       | 0xEC80
+
+### Terminating Element Data
+
+For `String Elements` and `UTF-8 Elements` the length of `Element Data` MAY be reduced by adding `Null Octets` to terminate the `Element Data` (see [the section on `Terminating Elements`](#terminating-elements)).
+
+In the following table, a four octet long `Element Data` is changed to a three octet long value followed by a `Null Octet`; the `Element Data Size` includes any `Null Octets` used to terminate `Element Data` so remains unchanged.
+
+| Status      | Element ID | Element Data Size | Element Data       |
+|-------------|------------|-------------------|--------------------|
+| Before edit | 0x3B4040   | 0x84              | 0x65626d6c         |
+| After edit  | 0x3B4040   | 0x84              | 0x6d6b7600         |
+
+Note that this method is NOT RECOMMENDED. For reductions of one octet, the method for `Extending the Element Data Size` SHOULD be used. For reduction by more than one octet, the method for `Adding a Void Element` SHOULD be used.
+
+## Considerations of Fixity when Updating Elements
```

Comment:
Is `Fixity` a term that should be defined somewhere ? I'm not sure most people know what it means.